test/performance/odp_ipsec.c

```diff
@@ -596,6 +596,14 @@ static uint8_t test_data[] = {
 	0x08, 0x00, 0xfb, 0x37, 0x12, 0x34, 0x00, 0x00
 };
 
+static inline void debug_packets(int debug, odp_packet_t *pkt, int num_pkts)
+{
+	if (odp_likely(!debug))
+		return;
```

Comment:
Is there an extra blank line here?

Reply:
I do not see any blank lines there. Or did you mean that one should be added?